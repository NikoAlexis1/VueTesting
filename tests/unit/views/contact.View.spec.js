import { shallowMount } from '@vue/test-utils'
import ContactView from '@/views/ContactView.vue'

describe('ContactView', () => {
    test('ValidaciÃ³n de match con el snapshot', () => {
        const wrapper = shallowMount(ContactView)
        expect(wrapper.html()).toMatchSnapshot()
    })
})