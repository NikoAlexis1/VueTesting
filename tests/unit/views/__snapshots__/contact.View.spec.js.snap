// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactView Validaci√≥n de match con el snapshot 1`] = `
<div>
  <h1>Contact</h1>
</div>
`;
